src/renderer/components/kube-object/kube-object-meta.tsx

```diff
@@ -22,10 +23,21 @@ export class KubeObjectMeta extends React.Component<KubeObjectMetaProps> {
     return hideFields.includes(field);
   }
 
+  renderObjectName() {
+    const { object } = this.props;
+    const statuses = kubeObjectStatusRegistry.getItemsForObject(object);
+
+    if (statuses.length === 0) {
+      return object.getName();
+    }
+
+    return <>{object.getName()} <KubeObjectStatusIcon key="icon" statuses={statuses} /></>;
+  }
```

Comment:
Better split it into 2 functions - `renderObjectName()` and `renderStatusIcon()`.